app/domain/authentication/authenticate.rb

```diff
@@ -11,10 +10,11 @@ module Authentication
 
   Authenticate ||= CommandClass.new(
     dependencies: {
-      token_factory:     TokenFactory.new,
-      validate_security: ::Authentication::Security::ValidateSecurity.new,
-      validate_origin:   ::Authentication::ValidateOrigin.new,
-      audit_log:         ::Audit.logger
+      token_factory:                       TokenFactory.new,
+      validate_webservice_is_whitelisted:  ::Authentication::Security::ValidateWebserviceIsWhitelisted.new,
+      validate_user_can_access_webservice: ::Authentication::Security::ValidateUserCanAccessWebservice.new,
```

Comment:
The word "user" is right here? are't hosts use authn?
or it's the same for both users and hosts?

Reply:
changed to `role`